tests/UI/pages/FO/home/index.js

```diff
@@ -16,6 +16,11 @@ class Home extends FOBasePage {
 
     this.pageTitle = global.INSTALL.SHOP_NAME;
 
+    // Selectors of slider
+    this.carouselSliderId = '#carousel';
+    this.carouselControlDirection = direction => `${this.carouselSliderId} a.${direction}.carousel-control`;
+    this.carouselSliderInner = `${this.carouselSliderId} ul.carousel-inner li`;
```

Comment:
Maybe 2 points Here : 

1. Name better your selectors
2. Add selector for `nth-child`

```suggestion
    this.carouselControlDirectionLink = direction => `${this.carouselSliderId} a.${direction}.carousel-control`;
    this.carouselSliderInnerList = `${this.carouselSliderId} ul.carousel-inner`;
    this.carouselSliderInnerListItems = `${this.carouselSliderInnerList} li`;
    this.carouselSliderInnerListItem = position => `${this.carouselSliderInnerListItems}:nth-child(${position})`;
```